provisions/roles/openshift/tasks/config.yml

```diff
@@ -31,7 +32,56 @@
       - "{{ origin_image_registry }}/{{ origin_image_name }}-docker-registry"
       - "{{ origin_image_registry }}/{{ origin_image_name }}-pod"
 
-- name: Create OpenShift origin container
+
+# Below is due to THIS issue: https://github.com/kubernetes/kubernetes/issues/43856 which we can't do anything about
+# (running Kubernetes docker-in-docker as well as on a CentOS 7 host causes issues...)
+# First we created the configuration so we can append some values
+- name: Create config OpenShift origin container
+  when: deployment == "docker"
+  docker_container:
+    name: origin-config
+    image: "{{ origin_image_registry }}/{{ origin_image_name }}:{{ origin_image_tag }}"
+    privileged: yes
+    pid_mode: host
+    network_mode: host
+    recreate: yes
+    volumes: "{{ openshift_volumes }}"
+    command: start --master=https://"{{ansible_default_ipv4.address}}":8443 --write-config=/var/lib/origin/openshift.local.config
+
+- name: Quick pause to catch up to configuration being propagated
+  when: deployment == "test"
+  pause:
+    seconds: 30
+
+# Append to the configuration to disable QOS
+# See: https://github.com/kubernetes/kubernetes/issues/43856
+- name: Append to config disabling QoS for OpenShift Origin
+  when: deployment == "docker"
+  lineinfile: 
+    dest: /var/lib/origin/openshift.local.config/node-{{ ansible_hostname }}/node-config.yaml
+    line: '{{ item }}'
+  with_items:
+    - 'kubeletArguments:'
+    - '  cgroups-per-qos:'
+    - '  - "false"' 
+    - '  enforce-node-allocatable:'
+    - '  - ""'
+
+- name: Create test OpenShift origin container
+  when: deployment == "docker"
+  docker_container:
+    name: origin
```

Comment:
should the name clash with actual "production OpenShift origin container" ([immediate next task](https://github.com/CentOS/container-pipeline-service/pull/492/files#diff-8b0f8f52a368116ae8035924236a6cb9R86))?

How about `name: origin_test`?